ci-operator/config/Azure/ARO-HCP/Azure-ARO-HCP-main__image-updater.yaml

```diff
@@ -0,0 +1,126 @@
+base_images:
+  prcreator:
+    name: prcreator
+    namespace: ci
+    tag: latest
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: ocp
+    tag: rhel-9-golang-1.24-openshift-4.20
+images:
+- dockerfile_literal: |
+    FROM src
+    RUN cat /etc/dnf/dnf.conf || true
+    RUN rpm --import https://packages.microsoft.com/keys/microsoft.asc
+    RUN dnf install -y https://packages.microsoft.com/config/rhel/9.0/packages-microsoft-prod.rpm
+    RUN ln -s /etc/yum.repos.d/microsoft-prod.repo /etc/yum.repos.art/ci/
+    RUN dnf install -y azure-cli libicu git
+    ADD prcreator /usr/bin/prcreator
+  from: src
+  inputs:
+    prcreator:
+      paths:
+      - destination_dir: .
+        source_path: /usr/bin/prcreator
+  to: image-updater-base
+resources:
+  '*':
+    requests:
+      cpu: 500m
+      memory: 1Gi
+tests:
+- as: image-updater-all-components
+  cron: 0 2 * * 1-5
+  steps:
+    test:
+    - as: update-images
+      commands: |
+        #!/bin/bash
+        set -o errexit
+        set -o nounset
+        set -o pipefail
+
+        echo "===== Configure Azure Authentication ====="
+        export AZURE_CLIENT_ID; AZURE_CLIENT_ID=$(cat /var/run/hcp-integration-credentials/client-id)
+        export AZURE_CLIENT_SECRET; AZURE_CLIENT_SECRET=$(cat /var/run/hcp-integration-credentials/client-secret)
+        export AZURE_TENANT_ID; AZURE_TENANT_ID=$(cat /var/run/hcp-integration-credentials/tenant)
+
+        echo "===== Sync Go workspace vendor directory ====="
+        go work vendor
+
+        # Configure git
+        git config user.name "openshift-ci-robot"
+        git config user.email "openshift-ci-robot@users.noreply.github.com"
+
+        echo "===== Building image-updater ====="
+        cd tooling/image-updater
+        make build
+
+        echo "===== Running image-updater for all components ====="
+        ./image-updater update --config config.yaml
+
+        # Check if there are any changes
+        if [[ $(git status --porcelain) == "" ]]; then
+          echo "No image updates needed for any components"
+          exit 0
+        fi
+
+        git add .
+        git commit -m "Ran image-updater for all components"
+
+        cd ../..
```

Comment:
This should run before the `git add .` as you are in the `tooling/image-updater`